frontend/lib/src/controllers/transaction_controller.dart

```diff
@@ -4,23 +4,37 @@ import 'package:frontend/src/models/transaction.dart';
 import 'package:frontend/src/services/transaction_service.dart';
 
 List<String> transactionTypes = ['Credit', 'Dept', 'Transfer'];
+List<String> transactionCategory = [
```

Comment:
change implementation to enum

`Foods/Drinks` is not valid as enum because of the `slash`
think of another term to encapsulate both and conditionally render `Foods/Drinks` based on that new enum value